homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -35,6 +45,7 @@
                         vol.Required(CONF_HOST): cv.string,
                         vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
                         vol.Optional(CONF_PORT, default=DEFAULT_PORT): cv.port,
+                        vol.Optional(CONF_LISTEN_PORT, default=DEFAULT_PORT): cv.port,
```

Comment:
We don't allow extending the config yaml. Configuration should be done in the config flow.

Reply:
Oh, ok. I guess we'll just leave that unchangeable for now and add an advanced flow later